### ISSUE TITLE
Please clarify the license

### ISSUE THREAD

**Issue by ind1go** [maxogden/nugget#22]
In the `package.json` it says "BSD". Could this be clarified to specify _which_ BSD, to make it easier for those doing license checks? Would BSD-3-Clause be too much of an assumption?

**Comment by bennymeg** [comment]
> In the package.json it says "BSD". Could this be clarified to specify which BSD, to make it easier for those doing license checks? 

I agree, but not only for license checks. Not including a licence file might imply that the project is closed source, and may lead to all sorts of problems.

> Would BSD-3-Clause be too much of an assumption?

I don't think we should assume that.

@maxogden  Please add a LICENCE file to your repository.

**Comment by ihbar** [comment]
I see that license file with BSD-3-Clause has been added in github, thanks a lot. 
It would be good to also update the package.json file with the information "license": "BSD-3-Clause", because BSD is not a valid SPDX identifier for npm.
Thanks
